charts/nginx-ingress/README.md

```diff
@@ -190,6 +193,131 @@ The command removes all the Kubernetes components associated with the release an
 Uninstalling the release does not remove the CRDs. To remove the CRDs, see [Uninstalling the
 CRDs](#uninstalling-the-crds).
 
+## Upgrading without downtime
+
+### Background
+
+In NGINX Ingress Controller version 3.1.0, [changes were introduced](https://github.com/nginxinc/kubernetes-ingress/pull/3606)
+to Helm resource names, labels and annotations to fit with Helm best practices.
+When using Helm to upgrade from a version prior to 3.1.0, certain resources like Deployment, DaemonSet and Service will
+be recreated due to the aforementioned changes, which will result in downtime.
+
+Although the advisory is to update all resources in accordance with new naming convention, to avoid the downtime
+please follow the steps listed in this page.
+
+### Upgrade Steps
+
+**Note** The following steps apply to both 2.x and 3.0.x releases.
+
+The steps you should follow depend on the Helm release name:
+
+{{<tabs>}}<!-- markdownlint-disable-line MD033 -->
```

Comment:
we can't have this in the markdown

Reply:
@lucacome this is blocking hugo custom tags, any other solution

Reply:
don't use tabs? 😄

Reply:
any other solution?

Reply:
Also, can you elaborate further why we can't have this in the markdown? Might help us find a workaround.

Reply:
I mean if you want to use tabs, no. Other than moving back to two separate documents and keeping them up to date manually.

Reply:
to clarify, this is only an issue in the README, not in the content inside `/docs/` ?

Reply:
The tab components should be removed from the README: they can instead be re-integrated as subheadings.

Reply:
the issue is that README is source of truth for helm website docs and everything gets automatically overwritten during release, to make exceptions for <<tabs>>, we'll have to edit that release script I think .